### ISSUE TITLE
Oppdaterte vurderinger etter lansering

### ISSUE THREAD

**Issue by SnorresKongesaga** [Artsdatabanken/assessments-frontend#1338]
En gammel klassiker #492 Kan du høste på nytt @eirikn-artsdatabanken ? Har gjort endringer på følgende vurderinger:

- Madrepora oculata, Siksakkkorall,Koralldyr
- Paragorgia arborea, sjøtre, Koralldyr
- Miota prolongata, Vepser
- Skeletocutis subincarnata, svellekjuke, sopper
- Acrosiphonia glacialis, Alger
- Geoglossum difforme, slimjordtunge, Sopper
- Epinotia nigricana, mørk kveldvikler, Sommerfugler
- Epinotia subsequana, grå vårkveldvikler, Sommerfugler

**Comment by eirikn-artsdatabanken** [comment]
Har oppdatert på hjemmesiden nå

**Comment by eirikn-artsdatabanken** [comment]
.. er usikker om det skal være nye revisjoner/versjoner - da må det kjøres et script i en database(?)

**Comment by SnorresKongesaga** [comment]
Ja, det er en noen nye revisjoner. F.eks., Epinotia nigricana her har jeg endret kategori, så det må inn en lenke til den forrige vurderingen. Se ramsløk hvis du lurer på hvordan det ser ut

**Comment by eirikn-artsdatabanken** [comment]
tror jeg må få instrukser fra @steinho på hvordan dette gjøres

**Comment by SnorresKongesaga** [comment]
Jeg har endret konklusjonen på [Cortinarius neofurvolaesus](https://artsdatabanken.no/lister/rodlisteforarter/2021/3934) 

Kan du høste på nytt @eirikn-artsdatabanken

**Comment by eirikn-artsdatabanken** [comment]
@SnorresKongesaga har oppdatert nå

**Comment by SnorresKongesaga** [comment]
Ser at den ikke kommer som en ny versjon med lenke til gammel versjon. Se her hvordan det skal se ut når jeg skriver tekst i det aktuelle tekstfeltet https://artsdatabanken.no/lister/rodlisteforarter/2021/25442 
Da står det "Gå til første utgave" med en lenke til originalen @eirikn-artsdatabanken

**Comment by eirikn-artsdatabanken** [comment]
se https://github.com/Artsdatabanken/assessments-frontend/issues/1338#issuecomment-1697325117

**Comment by SnorresKongesaga** [comment]
@steinho kan du se på dette og lage dokumentasjon på rutinen slik at også andre kan oppdatere vurderingene?

**Comment by SnorresKongesaga** [comment]
@eirikn-artsdatabanken  Har endret lappmeis https://artsdatabanken.no/lister/rodlisteforarter/2021/28205 
Under "Hvorfor er denne arten vurdert?" har jeg endret fra fremmedart til etablert i Norge

**Comment by eirikn-artsdatabanken** [comment]
> 28205

oppdatert på hjemmesiden nå

**Comment by eirikn-artsdatabanken** [comment]
> @eirikn-artsdatabanken Har endret lappmeis https://artsdatabanken.no/lister/rodlisteforarter/2021/28205 Under "Hvorfor er denne arten vurdert?" har jeg endret fra fremmedart til etablert i Norge

oppdatert på hjemmesiden nå

**Comment by SnorresKongesaga** [comment]
Jeg har endret to vurderinger @eirikn-artsdatabanken :

1. Rangifer tarandus, rein. Fikset skrivefeil "and" -> "og"
2. Ranunculus peltatus Schrank flyttet til Ranunculus schmalhausenii Luferov. **Puliseres som endring**

**Comment by SnorresKongesaga** [comment]
Hei @eirikn-artsdatabanken 
Jeg har endret vurderingen på disse to fra NE til EN, kan du oppdatere?
Ulmus glabra montana 
Ulmus glabra glabra

**Comment by SnorresKongesaga** [comment]
Hei @eirikn-artsdatabanken 
Jeg har endret på fire vurderinger, kan du oppdatere? Alle fire får endret kategori fra NE til hhv. EN, EN og NT, NT

Ulmus glabra montana, bergalm
Ulmus glabra glabra, skogalm
Carlina vulgaris vulgaris, småstjernetistel
Carlina vulgaris longifolia, storstjernetistel

Kan du også se på alm https://artsdatabanken.no/lister/rodlisteforarter/2021/23262 Jeg har rettet en skrivefeil, men det skal ikke stå "Justering etter publisering" på vurderingen

**Comment by SnorresKongesaga** [comment]
Hei @steinho 

Jeg har endret teksten på https://artsdatabanken.no/lister/rodlisteforarter/2021/572 Det står NT i teksten, har rett til VU. Kan du rulle ut?
